packages/backend/server/schema.prisma

```diff
@@ -397,6 +397,7 @@ model AiPrompt {
   // it is only used in the frontend and does not affect the backend
   action    String?  @db.VarChar
   model     String   @db.VarChar
+  optionalModels String[] @default([]) @db.VarChar
```

Comment:
```suggestion
  optionalModels String[] @default([]) @db.VarChar @map("optional_models")
```

Reply:
Thanks for your comments! Code modified.